vms/volunteer/models.py

```diff
@@ -79,7 +79,7 @@ class Volunteer(models.Model):
         blank=True)
 
     user = models.OneToOneField(User)
+    is_12_hours_format = models.BooleanField(default=True)
```

Comment:
This doesn't apply to Volunteers only but Admins as well so will need changes there as well